Update upgrade-hcx-azure-vmware-solutions.md (#120883)
Azure/azure-content: articles/azure-vmware/upgrade-hcx-azure-vmware-solutions.md

- As part of HCX update planning, and to ensure that HCX components are updated successfully, review the service update considerations and requirements. For planning HCX upgrade, see [Planning for HCX Updates](https://docs.vmware.com/en/VMware-HCX/4.5/hcx-user-guide/GUID-61F5CED2-C347-4A31-8ACB-A4553BFC62E3.html). 

- Ensure that you have a backup and snapshot of HCX connector in the on-premises environment, if applicable.
- For more information, see the [HCX support policy for legacy vSphere environment](https://kb.vmware.com/s/article/82702).

> - Check that you are using the [latest VMware HCX version validated with Azure VMware Solution](/introduction.md#vmware-software-versions).

Reviewer: ```suggestion
- Check that you are using the [latest VMware HCX version validated with Azure VMware Solution](introduction.md#vmware-software-versions).
```

> - Please see the [HCX support policy for legacy vSphere environment](./https://kb.vmware.com/s/article/82702).
> - Please note that the latest VMware HCX version validated with Azure VMware Solution is listed [here](./https://learn.microsoft.com/azure/azure-vmware/introduction#vmware-software-versions). Please check before upgrading HCX in your environment.

Reviewer: ```suggestion
- For more information, see the [HCX support policy for legacy vSphere environment](https://kb.vmware.com/s/article/82702).
- Check that you are using the [latest VMware HCX version validated with Azure VMware Solution](/introduction.md#vmware-software-versions).
```

MS style as well as fixing the links
